packages/server/lib/config.coffee

```diff
@@ -230,8 +230,9 @@ module.exports = {
     .value()
 
     if url = config.baseUrl
-      ## always strip trailing slashes
-      config.baseUrl = _.trimEnd(url, "/")
+      ## always strip trailing slashes and add one to end
```

Comment:
I'm not sure we should always be adding a slash anymore than we should always be removing one. Seems like the ideal is to leave the baseUrl as the user has it in their config, while also stripping extra `/`s. So `http://localhost:8080/` and `http://localhost:8080` stay the same and `http://localhost:8080///` becomes `http://localhost:8080/`.

Haven't tested this, but something like like `config.baseUrl = url.replace(/\/\\+$/, '/')` should work.

Reply:
Done: https://regexr.com/48rvt